### ISSUE TITLE
Support for Merged Cell in Tables

### ISSUE THREAD

**Issue by ccelestial** [karnov/htmltoword#49]
I have an this html tag

`<table align="center" border="1" cellpadding="0" cellspacing="0">`
`  <tbody>`
`    <tr>`
`      <td colspan="2">Lorem Ipsum with Colspan 2</td>`
`    </tr>`
`   <tr>`
`      <td><strong>Lorem Ipsum</td>`
`      <td><strong>Lorem Ipsum</td>`
`    </tr>`
`  </tbody>`
`</table>`

But in my document it appears to be like this.
<img width="701" alt="screen shot 2016-10-21 at 2 19 58 pm" src="https://cloud.githubusercontent.com/assets/5044679/19588629/9c66cac8-9799-11e6-954e-c65fd54bf7fb.png">

Does anyone have a solution for this?

**Comment by frankvilhelmsen** [comment]
the table seems to be fine,, maybe there is something external, css styles or otherwise.

**Comment by ccelestial** [comment]
I think the gem is not supporting the conversion of colspan of HTML to "w:vmerge" of WordprocessingML.

**Comment by ccelestial** [comment]
@frankvilhelmsen I edited the description and added a screenshot of what I'm getting in my pdf conversion.

**Comment by stats** [comment]
You should be look at [Tables.xslt](https://github.com/karnov/htmltoword/blob/master/lib/htmltoword/xslt/tables.xslt) which contains all of the code for styling tables. There is no vmerge or hmerge as part of the xslt. The tables code basically supports border properties and nothing else.

**Comment by ccelestial** [comment]
Hi @stats, that's also the file I referred that's why I saw that the gem aren't supporting vmerge or hmerge yet.

**Comment by lukelex** [comment]
@ccelestial I agree that this might be a good feature to have but since we don't use it we won't be investing time on it in the near future. BUT you're more than welcome to create a PR as we'll merge it.
